cmd/9ipfs/go.mod

```diff
@@ -0,0 +1,16 @@
+module harvey-os.org/cmd/9ipfs
```

Comment:
We should decide if we want to have go.mod files allowed per command, or if this should be unified with the top-level go.mod. In my professional work, we just maintain a single top-level go.mod per repo, but there may well be good reason to do it like this.

Reply:
ipfs has a lot of dependencies:
```
$ wc -l go.sum cmd/9ipfs/go.sum 
   101 go.sum
  1354 cmd/9ipfs/go.sum
```
so I wanted to keep it separate. Sub-modules are standard practice: see for example [gopls](https://github.com/golang/tools/blob/master/gopls/go.mod).